<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsColorRampLegendNodeWidgetBase</class>
 <widget class="QgsPanelWidget" name="QgsColorRampLegendNodeWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Legend Node Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="mUseContinuousLegendCheckBox">
     <property name="text">
      <string>Use continuous legend</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mLabelsGroup">
     <property name="title">
      <string>Labels</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QgsFilterLineEdit" name="mMaxLabelLineEdit"/>
      </item>
      <item row="6" column="1">
       <widget class="QgsFontButton" name="mFontButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Text format</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsFilterLineEdit" name="mMinLabelLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QgsFilterLineEdit" name="mPrefixLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mPrefixLabel">
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mSuffixLabel">
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mNumberFormatLabel">
        <property name="text">
         <string>Number format</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;(Applies to print layout legends only)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="mNumberFormatPushButton">
        <property name="text">
         <string>Customize</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsFilterLineEdit" name="mSuffixLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mLayoutGroup">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
      <item row="1" column="1">
       <widget class="QComboBox" name="mDirectionComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Orientation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mOrientationComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPanelWidget</class>
   <extends>QWidget</extends>
   <header>qgspanelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFontButton</class>
   <extends>QToolButton</extends>
   <header>qgsfontbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mPrefixLineEdit</tabstop>
  <tabstop>mSuffixLineEdit</tabstop>
  <tabstop>mMinLabelLineEdit</tabstop>
  <tabstop>mMaxLabelLineEdit</tabstop>
  <tabstop>mNumberFormatPushButton</tabstop>
  <tabstop>mFontButton</tabstop>
  <tabstop>mOrientationComboBox</tabstop>
  <tabstop>mDirectionComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
